code/modules/mob/living/carbon/human/species.dm

```diff
@@ -50,7 +50,9 @@
 	var/burnmod = 1		// multiplier for burn damage
 	var/coldmod = 1		// multiplier for cold damage
 	var/heatmod = 1		// multiplier for heat damage
-	var/punchmod = 0	// adds to the punch damage
+	var/punchdamagelow = 0       //lowest possible punch damage
+	var/punchdamagehigh = 9      //highest possible punch damage
+	var/punchstunthreshold = 9//damage at which punches from this race will stun
```

Comment:
no it wouldn't, the value would still be 9, leaving the outcome unchanged.
it would simple be ```H.dna...``` instead of ```M.dna...```